content/v2.0/reference/cli/influx/secret/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: influx secret – Manage secrets
+description: The 'influx secret' command manages secrets.
+menu:
+  v2_0_ref:
+    name: influx secret
+    parent: influx
+weight: 101
+v2.0/tags: [secrets]
+---
+
+The `influx secret` command manages secrets.
+
+## Usage
+```
+influx secret [flags]
+influx secret [command]
+```
+
+## Subcommands
+| Subcommand                                          | Description            |
+|:----------                                          |:-----------            |
+| [delete](/v2.0/reference/cli/influx/secret/delete/) | Delete a secret        |
+| [find](/v2.0/reference/cli/influx/secret/find/)     | Find secrets           |
```

Comment:
List secrets or Find a secret?

Reply:
Good catch. The tools team is actually changing `find` to `list` across all CLI commands. My plan is to update this in another PR when that change is merged. See https://github.com/influxdata/docs-v2/issues/753.